db/schema.rb

```diff
@@ -287,6 +287,12 @@
     t.string "case_status"
     t.integer "assigned_user"
     t.boolean "continuous_exposure", default: false
+    t.datetime "latest_assessment_at"
+    t.datetime "latest_fever_or_fever_reducer_at"
+    t.date "latest_positive_lab_at"
```

Comment:
it's because it's consistent with and based on the `specimen_collection` date in `laboratories`, which is also a date: https://github.com/SaraAlert/SaraAlert/blob/SARAALERT-483/db/schema.rb#L83

I actually kind of wish they were both `datetime` instead of `date` but it's too late for that now